python/lsst/ts/lasertracker/laser_tracker_csc.py

```diff
@@ -667,6 +645,32 @@ async def measure_alignment(self, target: str) -> None:
             target_offset["dRX"] -= self.config.zero_points["camera"]["u"]
             target_offset["dRY"] -= self.config.zero_points["camera"]["v"]
 
+        # Add LUT
+
+        # compensation class for CAM LUT offsets
+        self.compensation = compensation.Compensation(
```

Comment:
`compensation` is only used in this method and not needed at the class level. You can remove the `self.` and just keep `compensation` a local variable. 

Also, you hard coded the target to `CAM` so other targets would not be covered, e.g., `M2`.

Make sure you use `target` to retrieve the LUT.